indico/web/templates/_session_bar.html

```diff
@@ -171,40 +171,45 @@
         {%- if ambiguous and territory %} ({{ territory }}){% endif -%}
     {%- endset %}
     {% if lightweight %}
+        {# FIXME: [A11Y] Replace with button with appropriate state (requires new dropdown impl.) #}
         <a id="language-selector-link"
-           href=""
+           href="#language-list"
+           data-ignore-href="true"
            class="{{ 'arrow' if can_choose }} js-dropdown" {% if can_choose %}data-toggle="dropdown"{% endif %}>
             {%- trans %}My language ({{ language }}){% endtrans %}
         </a>
     {% else %}
         <button id="language-selector-link" class="icon-earth i-button {{ 'arrow js-dropdown' if can_choose }}"
-                {% if can_choose %}data-toggle="dropdown"{% else %}disabled{% endif %}>
+           {% if can_choose %}data-toggle="dropdown"{% else %}disabled{% endif %}
+           aria-label="{% trans %}The current language is {{ language }}. Use a different language.{% endtrans %}">
             {{- language -}}
         </button>
     {% endif %}
     {% if can_choose %}
-        <ul class="i-dropdown">
+        <ul id="language-list" class="i-dropdown">
             {% for name, (title, territory, __) in languages|dictsort(by='value') %}
+                {% set html_lang = name.replace('_', '-') %}
                 <li>
                     {% if force_locale and name not in force_locale_alts %}
-                        <a class="disabled" title="{% trans %}This language is not available here.{% endtrans %}">
+                        <button disabled data-qtip-oldtitle="{% trans %}This language is not available here.{% endtrans %}" lang="{{ html_lang }}">
```

Comment:
Indeed it does not. I'll figure it out.

Reply:
assuming you tested with `title="..."` and it didn't work, this is probably because disabled elements do not receive events.

a possible solution could be going back to `a` elements and just adding `href="#"` - as long as the default event is prevented hat wouldn't be an issue, and they'd hopefully be accessible nonetheless.

putting the title on the `li` could also be an option

Reply:
So it doesn't work with `disabled` buttons, which is, in fact, correct. I'll keep this as a `title` attribute in order to keep the translation text, and it doesn't really hurt. Since these buttons are not clickable anyway it should be fine.

The other disabled button, for the unavailable language may need additional explanation. I'm not sure the one provided in the tooltip is sufficient, though. To me it is still not clear why it is unavailable and what the user can do to rectify it. May be better to simply not render such languages to begin with?

Reply:
"unavailable languages" are part of the 'force event languages' feature where an event can specify that it wants to allow the indico UI to only be shown in certain languages, e.g. because of event organizers who do not want mixed language content.

you can test this by creating an event and on the main event settings page setting a specific event language and toggling the 'force' switch:

![image](https://github.com/indico/indico/assets/179599/b0ea65db-bc23-42c1-9a21-ada28bc2cc03)

the idea of showing the disabled languages is so a user knows why the language the may be using indico in at other times is suddenly not available.

Reply:
> disabled elements do not receive events.

And they are explicitly filtered out in the tooltip code.


> the idea of showing the disabled languages is so a user knows why the language the may be using indico in at other times is suddenly not available.

What do you think about the following idea. Once we implement the new pull-down menu (with larger surface area) we could express this by saying:

"The following languages are disabled for this event:"